_data/pricing-table.yml

```diff
@@ -0,0 +1,125 @@
+- title: Gruntwork Landing Zone
+  tooltip: 
+  rows:
+    - row: 'Terraform account baselines'
+      tooltip: Automatically apply security baselines, defined in Terraform, to all your AWS accounts, ensuring that all of your accounts are properly configured with AWS CloudTrail, AWS Config, AWS Config rules, Amazon GuardDuty, Macie, IAM roles, IAM Access Analyzer, VPCs, and more.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Monitoring: CloudTrail, AWS Config'
+      tooltip: Configure monitoring and audit logging for all your AWS accounts using CloudTrail and AWS Config.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Networking: VPCs, subnets, etc'
+      tooltip: Set up best-practice networking infrastructure for all of your AWS accounts, including VPCs, subnets, route tables, Internet Gateways, NAT Gateways, VPC endpoints, NACLs, and more.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Scanning: GuardDuty, Macie'
+      tooltip: Configure security scanning for all your AWS accounts using GuardDuty and Macie.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'IAM: users, roles, groups'
+      tooltip: Use IAM users, roles, and groups for authentication and authorization in your AWS accounts.
+      standard: 'yes'
+      enterprise: 'yes'
+    - row: 'Control Tower integration'
```

Comment:
Can you add a beta label to this? I'm trying to spot all the places it shows up and make sure we're consistent 😄